src/strategies/Strategy.ts

```diff
@@ -48,8 +48,12 @@ class Strategy {
 
   async toJSON (): Promise<StrategyJSON> {
     const orders = await Promise.all(
-      this.orders.map(async order => await order.toJSON())
-    )
+      this.orders.map(async (order, orderIndex) => ( 
+        {
+          ...await order.toJSON(),
+          orderIndex,
```

Comment:
we might want this to be named `index`, so it's `order.index` instead of `order.orderIndex`